### PR TITLE
Create AcronymApostrophe.yml

### PR DESCRIPTION
**Pull request by alexvonme**
I added a rule to check if any acronyms received an apostrophe.

It currently works properly, but I concede that both the message and token could be better written. Please feel free to suggest edits.

### CODE REVIEW

**Review by ccoVeille — APPROVED** [review]
LGTM 👍

The only problem I see with vale rules is somehow the inability to define unit tests for them, and something that could be added to a CI workflow

File: common/AcronymApostrophe.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+extends: existence
+message: "The acronym [%s] should not be in possesive form or used with an apostrophe in plural form."
+level: error
+ignorecase: true
+tokens:
+  - ^[A-Z]{2,}'(?:s|es|ies)
```
Comment (ccoVeille) [inline common/AcronymApostrophe.yml:6]:
```suggestion
  - ^[A-Z]{2,}'(?:s|es|ies)?
```

Maybe this to catch:
- invalid apostrophe: `USA' army`
- acronym ending with S, that may be used like this: `ISS' members`

File: common/AcronymApostrophe.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+extends: existence
+message: "The acronym [%s] should not be in possesive form or used with an apostrophe in plural form."
+level: error
+ignorecase: true
```
Comment (ccoVeille) [inline common/AcronymApostrophe.yml:4]: The ignore case seems a problem to me

Won't you catch lower case top?

**Review by ccoVeille — CHANGES_REQUESTED** [review]

File: common/AcronymApostrophe.yml (hunk 2 above)
Comment (alexvonme) [reply common/AcronymApostrophe.yml:4]: Yeah, thanks. I think I got too used to writing that down for other rules.

File: common/AcronymApostrophe.yml (hunk 1 above)
Comment (alexvonme) [reply common/AcronymApostrophe.yml]: Good idea, I've added this functionality too.

**Comment by alexvonme** [comment]
Thank you very much @ccoVeille, I've updated the pull request.

**Pull request merged**